Cargo.toml

```diff
@@ -3,11 +3,12 @@ name = "tauri-plugin-sql"
 version = "0.0.0"
 authors = ["Tauri Programme within The Commons Conservancy"]
 description = "A Tauri plugin for easily interfacing with SQL databases"
-edition = "2018"
+license = "Apache-2.0 OR MIT"
+edition = "2021"
 exclude = ["/examples", "/webview-dist", "/webview-src", "node_modules"]
 
 [dependencies]
-tauri = "1.0.0-beta.8"
+tauri = { git = "https://github.com/tauri-apps/tauri", branch = "next" }
```

Comment:
Can we keep this on beta.8 for now? This plugin is pretty much the most used one and idk if breaking this is a good idea right now 🤔 
Same for the examples then ofc

Reply:
Fine by me too, my intention was to get all plugins to a similar state and that includes using the latest tauri version.
Buuuut I guess you're right, we've told people to use this plugin already and they might not be happy with that (event though it's technically 0.x)

Reply:
reverted!